pytest_astropy_header/display.py

```diff
@@ -71,14 +68,13 @@ def pytest_report_header(config):
     args = config.args
 
     # TESTED_VERSIONS can contain the affiliated package version, too
-    if len(TESTED_VERSIONS) > 1:
-        for pkg, version in TESTED_VERSIONS.items():
-            if pkg not in ['Astropy']:
-                s = "\nRunning tests with {} version {}.\n".format(
-                    pkg, version)
-    else:
-        s = "\nRunning tests with Astropy version {}.\n".format(
-            TESTED_VERSIONS['Astropy'])
+    skip_astropy = len(TESTED_VERSIONS) > 1 and 'Astropy' in TESTED_VERSIONS
```

Comment:
Do we even need the `len` check? What about lowercase `astropy`?
```suggestion
    skip_astropy = 'Astropy' in TESTED_VERSIONS
```